blues/airnote.live
# bug: fix broken selectFields body values

# Problem Context

A Notehub change that allows users to selectively fetch only certain fields from an event was updated, and a previously working query fetching the entire `body` in an event stopped working.

## Changes

* Refine the Notehub query to selectively get all the fields within the `body` object required by the Airnote dashboard.
* Update the automated tests to match the changes caused by switching from fetching data via an undocumented Notehub API to the Notehub JS SDK's API instead.

## Screenshot (if applicable)

Prod Airnote dashboard (broken)
![Screenshot 2024-09-26 at 1 46 52 PM](https://github.com/user-attachments/assets/2fb79459-da48-4b5b-8a33-af5e4c52df9e)

Local Airnote dashboard (fixed) 
![Screenshot 2024-09-26 at 1 46 24 PM](https://github.com/user-attachments/assets/af8aafdc-47a0-4be7-8719-d9dcd8f5acee)

## Testing

Unit / integration / e2e tests?

All unit and e2e tests pass.

Steps to test manually?

1. Go to a live Airnote dashboard in prod like: https://airnote.live/dev:864475044215258/dashboard
2. See that all the data is NaN values
3. Go to that same dashboard in local: http://localhost:5173/dev:864475044215258/dashboard
4. See that all the data is present and accounted for 

Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

https://bluesinc.atlassian.net/jira/software/c/projects/BLUESDEV/boards/13?selectedIssue=BLUESDEV-424
